src/ontology/mondo-ingest.Makefile

```diff
@@ -441,6 +441,7 @@ lexical-matches: $(MAPPINGSDIR)/mondo-sources-all-lexical.sssom.tsv
 ###################################
 #### Lexmatch-SSSOM-compare #######
 ###################################
+# This goal also creates, for all sources: unmapped_SOURCE_lex.tsv, unmapped_SOURCE_lex_exact.tsv
 lexmatch/README.md: $(SCRIPTSDIR)/lexmatch-sssom-compare.py $(MAPPINGSDIR)/mondo-sources-all-lexical.sssom.tsv $(ALL_EXCLUSION_FILES)
```

Comment:
## Clarifying comment
Unrelated to the bug fix at hand, but related to the lexmatch pipeline.
The goal `lexmatch/README.md` is confusing and against the typical way that goals in `make` are normally supposed to work `mondo-ingest`, in that it only shows 1 target `lexmatch/README.md`, even though it generates a number of files.

I can see why this was done. We'd have to list 2 output files `*_lex.tsv` and `*_lex_exact.tsv` for each source. But then, if someone wanted to run on a single source, and they ran, say, `unmapped_icd11foundation_lex.tsv`, it would run for ALL sources.

This could be addressed with a refactor to make this goal and the Python file that it call to be in line with how we typically have goals / pipelines set up in `mondo-ingest`--that is, the target of the goal would be a wildcard, as in `slurp/%.tsv`, and then to run them all, we'd have separate goal for that, e.g. `slurp-all`.

For now, I've added a clarifying comment so that if someone is searching for the goal for these outputs, they can easily find it.

Reply:
FYI @twhetzel I also noticed there are several issues related to `lexmatch/README.md`, so I updated one of those issue into a grouping issue:
- #516